packages/fluent/scss/listview/_theme.scss

```diff
@@ -8,7 +8,11 @@
             var( --kendo-listview-text, $kendo-listview-text ),
             var( --kendo-listview-bg, $kendo-listview-bg ),
             var( --kendo-listview-border, $kendo-listview-border )
-        )
+        );
+
+        &.k-disabled {
+            color: var( --kendo-listview-disabled-text, $kendo-listview-disabled-text );
+        }
```

Comment:
These styles are not needed. The ListView component API does not provide a way to disable the component.